<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodeVerification</class>
 <widget class="QDialog" name="CodeVerification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Email Verification</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    /* Main Dialog Styling */
    QDialog {
        background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                    stop:0 #1e3c72, stop:1 #2a5298);
        color: #ffffff;
    }
    
    /* Title Label */
    QLabel#titleLabel {
        color: #ffffff;
        font-family: 'Segoe UI', Arial, sans-serif;
        font-size: 24px;
        font-weight: 700;
        text-align: center;
        background: transparent;
        border: none;
        text-shadow: 2px 2px 4px rgba(0,0,0,0.3);
    }
    
    /* Instruction Label */
    QLabel#instructionLabel {
        color: rgba(255, 255, 255, 0.9);
        font-family: 'Segoe UI', Arial, sans-serif;
        font-size: 14px;
        font-weight: 400;
        text-align: center;
        background: transparent;
        border: none;
    }
    
    /* Container Frame */
    QFrame#containerFrame {
        background: rgba(255, 255, 255, 0.05);
        border: 1px solid rgba(255, 255, 255, 0.1);
        border-radius: 20px;
    }
    
    /* Line Edit Fields */
    QLineEdit {
        background: rgba(255, 255, 255, 0.9);
        border: 2px solid rgba(255, 255, 255, 0.3);
        border-radius: 12px;
        color: #2c3e50;
        font-family: 'Segoe UI', Arial, sans-serif;
        font-size: 16px;
        font-weight: 500;
        padding: 15px 20px;
        selection-background-color: #4facfe;
        selection-color: white;
        text-align: center;
        letter-spacing: 3px;
    }
    
    QLineEdit:focus {
        border: 2px solid #4facfe;
        background: rgba(255, 255, 255, 0.95);
        box-shadow: 0 0 20px rgba(79, 172, 254, 0.4);
    }
    
    QLineEdit:hover {
        border: 2px solid rgba(79, 172, 254, 0.5);
        background: rgba(255, 255, 255, 0.95);
    }
    
    /* Placeholder text color */
    QLineEdit::placeholder {
        color: #7f8c8d;
        font-style: italic;
        letter-spacing: 1px;
    }
    
    /* Professional Buttons */
    QPushButton {
        background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                    stop:0 #4facfe, stop:1 #00f2fe);
        border: none;
        border-radius: 12px;
        color: #ffffff;
        font-family: 'Segoe UI', Arial, sans-serif;
        font-size: 16px;
        font-weight: 600;
        padding: 12px 24px;
        text-shadow: 1px 1px 2px rgba(0,0,0,0.3);
        box-shadow: 0 4px 15px rgba(79, 172, 254, 0.3);
    }
    
    QPushButton:hover {
        background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                    stop:0 #5fbdff, stop:1 #1af3fe);
        transform: translateY(-2px);
        box-shadow: 0 6px 20px rgba(79, 172, 254, 0.4);
    }
    
    QPushButton:pressed {
        background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                    stop:0 #3a9bfd, stop:1 #00d9fe);
        transform: translateY(1px);
        box-shadow: 0 2px 10px rgba(79, 172, 254, 0.2);
    }
    
    /* Resend Button Special Styling */
    QPushButton#resendButton {
        background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                    stop:0 #667eea, stop:1 #764ba2);
        box-shadow: 0 4px 15px rgba(102, 126, 234, 0.3);
    }
    
    QPushButton#resendButton:hover {
        background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                    stop:0 #7b8beb, stop:1 #8659b3);
        box-shadow: 0 6px 20px rgba(102, 126, 234, 0.4);
    }
    
    QPushButton#resendButton:pressed {
        background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                    stop:0 #5a6fd8, stop:1 #6a4191);
        box-shadow: 0 2px 10px rgba(102, 126, 234, 0.2);
    }
    
    /* Cancel Button Special Styling */
    QPushButton#cancelButton {
        background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                    stop:0 #ff6b6b, stop:1 #ee5a5a);
        box-shadow: 0 4px 15px rgba(255, 107, 107, 0.3);
    }
    
    QPushButton#cancelButton:hover {
        background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                    stop:0 #ff7979, stop:1 #f56565);
        box-shadow: 0 6px 20px rgba(255, 107, 107, 0.4);
    }
    
    QPushButton#cancelButton:pressed {
        background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                    stop:0 #ff5757, stop:1 #e74c3c);
        box-shadow: 0 2px 10px rgba(255, 107, 107, 0.2);
    }
   </string>
  </property>
  <widget class="QFrame" name="containerFrame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>390</width>
     <height>320</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>330</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Email Verification</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="instructionLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>75</y>
      <width>330</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>We've sent a verification code to your email.
Please enter the code below to continue.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="codeLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>250</width>
      <height>60</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>6</number>
    </property>
    <property name="placeholderText">
     <string>Enter 6-digit code</string>
    </property>
   </widget>
   <widget class="QPushButton" name="verifyButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
      <fontweight>DemiBold</fontweight>
     </font>
    </property>
    <property name="toolTip">
     <string>Click to verify the code</string>
    </property>
    <property name="text">
     <string>Verify Code</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resendButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
      <fontweight>DemiBold</fontweight>
     </font>
    </property>
    <property name="toolTip">
     <string>Resend verification code</string>
    </property>
    <property name="text">
     <string>Resend Code</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>270</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
      <fontweight>DemiBold</fontweight>
     </font>
    </property>
    <property name="toolTip">
     <string>Cancel verification</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
